src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+$main-color: #060b35;
+$btn-color: #00acdc;
+$white-bg-color: #fff;
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
You should not hardcode height. It should be calculated by your elements height and margins.